src/components/SearchForm/SearchForm.jsx

```diff
@@ -25,17 +18,31 @@ const SearchForm = () => {
         <label htmlFor='search' className='search-form__field'>
           <input
             type='search'
-            className='search-form__input'
+            className={
+              props.isSearchFieldEmpty
+                ? 'search-form__input search-form__input_placeholder_red'
+                : 'search-form__input'
+            }
             id='search-input'
             name='search'
-            placeholder='Enter topic'
+            placeholder={
+              !props.isSearchFieldEmpty
+                ? 'Enter topic'
+                : 'Please enter the keywords'
+            }
+            required
+            value={inputs.search || ''}
+            onChange={handleChange}
           ></input>
         </label>
 
         <Button
           className='search-form__submit-button'
           type='submit'
           title='Search'
+          onClick={(event) =>
```

Comment:
` onClick={(event) =>`


please, delete it from the button


Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses